cdm/core/src/main/java/thredds/client/catalog/builder/CatalogBuilder.java

```diff
@@ -59,12 +59,28 @@ public Catalog buildFromLocation(String location, URI baseURI) {
     }
 
     this.baseURI = baseURI;
+    try {
+      validatePort(baseURI);
+    } catch (IllegalArgumentException e) {
+      errlog.format("Invalid port number = '%s' err='%s'%n ", baseURI.toASCIIString(), e.getMessage());
```

Comment:
I think this will end up returning port information in the browser error message...

Reply:
It should be OK, as it's not giving away any sensitive info. It's send this if they try a root-privileged port that isn't 80 or 443... which is probably deliberate attempt at shenanigans.

Reply:
works for me